### PR TITLE
updates in Logs&Events page

### PR DESCRIPTION
**Pull request by nolkhova**

### CODE REVIEW

File: docs/monitoring/logsandevents/logsandevents.md [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
-The logs and events are visible within AxonOps dashboard allowing quick access to them without having to log in to the individual servers.
 
-![](./0.JPG)
+AxonOps provides a powerful logging feature that allows you to search and filter logs based on different parameters such as DC/Rack/Node, Log Level, Event Type, Source and Log Content.
 
-For a free text search fulfill the ***content*** input.
+![](logsUI.png)
 
-You can search by regex expression via the `/<expression>/` pattern.
+The logs and events are visible within AxonOps dashboard and Logs & Events tab allowing quick access to them without having to log in to the individual servers.
```
Comment (rgooding) [inline docs/monitoring/logsandevents/logsandevents.md:8]: Could you please change "log in" to "login" here, I know it doesn't really matter but since this page is about logs it will reduce any possible confusion for the reader

File: docs/monitoring/logsandevents/logsandevents.md [hunk 2]
```diff
@@ -1,11 +1,42 @@
...
\ No newline at end of file
+#### Logs Source and Event Type
+
+You can filter logs based on the log source (cassandra, axon-server and axon-agent logs) and event type to narrow down search results.
+
+![](logs_event_type.png)
+
+#### Search by Content
+
+For a free text search fulfill the ***content*** input with a keyword or search by regex expression via the `/<expression>/` pattern.
```
Comment (rgooding) [inline docs/monitoring/logsandevents/logsandevents.md:18]: I would rephrase this, maybe something like this: "For a free text search enter a keyword in the **content** input or use the `/<expression>/` syntax to search by regular expression."

File: docs/monitoring/logsandevents/logsandevents.md [hunk 3]
```diff
@@ -1,11 +1,42 @@
...
+
+![](logs_example_regexp2.png)
+
+* Display logs that contain both patterns in a line, in this case "Segment" and "deleted:
+
+![](logs_example_regexp3.png)
+
+
+#### Search by Log Level
+
```
Comment (rgooding) [inline docs/monitoring/logsandevents/logsandevents.md:36]: This feels a little stuck on the end and out of place. Maybe it needs a sentence or two here explaining about searching by log level and maybe also a screenshot? I also think the instructions for setting up the debug level should be under a subheading so it appears in the page navigation on the right hand side

**Review by rgooding — CHANGES_REQUESTED** [review]
In addition to my comments below could you see if you can do something about the logsUI.png image? It doesn't display very well scaled down to fit in the page so maybe you need to take a screenshot of it with the window resized a bit smaller.

File: docs/monitoring/logsandevents/logsandevents.md (hunk 1 above)
Comment (nolkhova) [reply docs/monitoring/logsandevents/logsandevents.md]: changed

File: docs/monitoring/logsandevents/logsandevents.md (hunk 2 above)
Comment (nolkhova) [reply docs/monitoring/logsandevents/logsandevents.md]: changed

File: docs/monitoring/logsandevents/logsandevents.md (hunk 3 above)
Comment (nolkhova) [reply docs/monitoring/logsandevents/logsandevents.md]: done. I changed the heading order as well.

**Review by rgooding — APPROVED** [review]
